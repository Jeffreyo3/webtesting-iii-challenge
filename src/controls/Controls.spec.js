// Test away!
import React from 'react';
import renderer from 'react-test-renderer';
import * as rtl from "@testing-library/react";
import "@testing-library/jest-dom/extend-expect";

import Controls from './Controls';

