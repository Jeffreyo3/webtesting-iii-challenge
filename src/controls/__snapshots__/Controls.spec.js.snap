// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Controls /> snapshop 1`] = `
<DocumentFragment>
  <div
    class="controls panel"
  >
    <button
      class="toggle-btn"
      data-testid="lockButton"
      disabled=""
    >
      Lock Gate
    </button>
    <button
      class="toggle-btn"
      data-testid="openButton"
    >
      Close Gate
    </button>
  </div>
</DocumentFragment>
`;
